static-build/qemu-virtiofs/Dockerfile

```diff
@@ -0,0 +1,44 @@
+from ubuntu:18.04
+
+ARG QEMU_VIRTIOFS_REPO
+# commit/tag/branch
+ARG QEMU_VIRTIOFS_TAG
+ARG PREFIX
```

Comment:
I think we will need something like the following as well: https://github.com/kata-containers/packaging/commit/255bae15150ce382d769a9d7866f9bee07e6f742